api/Cache.json

```diff
@@ -4,57 +4,57 @@
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/Cache",
         "support": {
+          "webview_android": {
+            "version_added": "43",
+            "notes": [
+              "From 40 to 42, this was only available on service workers."
+            ]
```

Comment:
Seems a bit strange to have all notes in arrays, even if there is only one note. Make these string types?

Reply:
I thought it was required. Sorry.